### DISCUSSION TITLE
farasa segmentation

### DISCUSSION THREAD

**Discussion by yassmine-lam** [aub-mind/arabert#174] [Q&A] [answered]
Hi,

Based on your experience, which model is better for Named entity recognition and why: a BERT model with or without farasa segmentation. Does segmentation affect sequence labeling tasks?

Many thanks

**Comment by WissamAntoun** [comment, marked as answer]
for NER i would go for a BERT without segmentation.

in our paper we experimented with both, and found out that segmentation was hurting performance. We guessed that it is caused by the label being put on the "ال" (or other prefixes) at the beginning of most words.

**Comment by yassmine-lam** [comment]
Ok. Thank u very much for ur answer
